remoting/etcdv3/listener_test.go

```diff
@@ -18,17 +18,24 @@
 package etcdv3
 
 import (
+	"net/url"
+	"os"
+	"testing"
 	"time"
 )
 
 import (
+	"github.com/coreos/etcd/embed"
+	gxetcd "github.com/dubbogo/gost/database/kv/etcd/v3"
 	"github.com/stretchr/testify/assert"
 )
 
 import (
 	"github.com/apache/dubbo-go/remoting"
 )
 
+const defaultEtcdV3WorkDir = "/tmp/default-dubbo-go-remote.etcd"
```

Comment:
will it cause problem in docker env? like access deny , can not open file etc?

Reply:
May be is ok, i just use the old dir in database/kv/etcd/v3/client_test.go, so should be ok.